src/components/widgets/AltDateWidget.js

```diff
@@ -63,6 +64,12 @@ class AltDateWidget extends Component {
     });
   };
 
+  onBlur = () => {
+    if (this.props.onBlur){
+      this.props.onBlur(toDateString(this.state, this.props.time));
+    }
+  };
```

Comment:
Don't we want to trigger the event when data are consistent, the same way it's done in `onChange`?